Sources/Extensions/UIView+Constraints.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Constraints.swift
+//  Keychain
+//
+//  Created by Jelle Vandebeeck on 11/02/2018.
+//
+
+import UIKit
+
+extension UIView {
+    
+    public func constraint(to view: UIView) {
```

Comment:
🎈would name this with a verb
`func constrain(to view: UIView)` or `func constrainToEdgesOf(_ view: UIView)`

Reply:
Or `func constrainToEdges(of view: UIView)`?